modules/material/themes/material/mfa/prompt-for-mfa-webauthn.twig

```diff
@@ -9,6 +9,19 @@
 
   <script>
     document.addEventListener('DOMContentLoaded', function() {
+      const rememberMeCheckbox = document.getElementById('rememberMe');
+      
+      if (rememberMeCheckbox) {
+        const desiredRememberMeState = localStorage.getItem('desiredRememberMeState');
+        if (desiredRememberMeState === 'true') {
+          rememberMeCheckbox.checked = true;
+        }
+
+        rememberMeCheckbox.addEventListener('change', function() {
+          localStorage.setItem('desiredRememberMeState', rememberMeCheckbox.checked);
+        });
+      }
+      
```

Comment:
I feel like 4x is too much to duplicate this code. Could you drop a new js file in the modules/mfa/public folder and reference it like `<script src="{{ remember_me_js_path }}"></script>`, similar to the browser.js file?

Reply:
Yeah, not sure why I didn't do that to start with.